runtime/doc/options.txt

```diff
@@ -4490,10 +4480,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	option may be relative or absolute.
 	- Use commas to separate directory names: >
 		:set path=.,/usr/local/include,/usr/include
-<	- Spaces can also be used to separate directory names (for backwards
-	  compatibility with version 3.0).  To have a space in a directory
-	  name, precede it with an extra backslash, and escape the space: >
-		:set path=.,/dir/with\\\ space
```

Comment:
Would be good to replace this with a blurb about how to have spaces in a directory name

Reply:
I've changed the code so spaces just work as normal.

Reply:
May still be worth mentioning here? Or at least in the `news.txt` entry.

Reply:
And maybe add a test for correctly handling dirs with spaces?

Reply:
There aren't really any existing tests for backupdir using multiple values.

Reply:
(I meant single value, but one that includes spaces)